pkg/driver/node.go

```diff
@@ -122,18 +124,24 @@ func (ns *node) waitForVolumeToBeReady(volID string) (*jv.JivaVolume, error) {
 		retry++
 		if instance.Status.Phase == jv.JivaVolumePhaseReady && instance.Status.Status == "RW" {
 			return instance, nil
-		} else if retry > MaxRetryCount {
+		} else if retry <= MaxRetryCount {
+			sleepInterval = 30
```

Comment:
Should we reduce this retry count to 2/3/5 seconds and increase the default MaxRetryCount

Reply:
done